### PR TITLE
[4.1] RavenDB-11116 - Error getting license support info

### PR DESCRIPTION
**Pull request by grisha-kotler**
http://issues.hibernatingrhinos.com/issue/RavenDB-11116

### CODE REVIEW

File: src/Raven.Server/Commercial/LicenseManager.cs [hunk 1]
```diff
@@ -129,7 +130,7 @@ public void Initialize(StorageEnvironment environment, TransactionContextPool co
             {
                 _leaseLicenseTimer = new Timer(state =>
                     AsyncHelpers.RunSync(ExecuteTasks), null,
-                    (int)TimeSpan.FromMinutes(1).TotalMilliseconds,
+                    (int)TimeSpan.FromMinutes(0).TotalMilliseconds,
```
Comment (ayende) [inline src/Raven.Server/Commercial/LicenseManager.cs]: Is this intentional, and if so , why?

File: src/Raven.Server/Commercial/LicenseManager.cs [hunk 2]
```diff
@@ -129,7 +130,7 @@ public void Initialize(StorageEnvironment environment, TransactionContextPool co
             {
                 _leaseLicenseTimer = new Timer(state =>
                     AsyncHelpers.RunSync(ExecuteTasks), null,
```
Comment (ayende) [inline src/Raven.Server/Commercial/LicenseManager.cs]: Who handles errors here?

File: src/Raven.Server/Commercial/LicenseManager.cs (hunk 1 above)
Comment (grisha-kotler) [reply src/Raven.Server/Commercial/LicenseManager.cs]: no, i reverted this change

File: src/Raven.Server/Commercial/LicenseManager.cs (hunk 2 above)
Comment (grisha-kotler) [reply src/Raven.Server/Commercial/LicenseManager.cs]: it is handled in ExecuteTasks

**Pull request merged**